etc/kayobe/pulp.yml

```diff
@@ -317,6 +322,106 @@ stackhpc_pulp_repository_rpm_repos:
     state: present
     required: "{{ stackhpc_pulp_sync_el_8 }}"
 
+  # Additional repositories required to build Kolla containers from local Pulp service.
+  - name: CentOS Stream 8 - OpenStack Wallaby
+    url: "{{ stackhpc_release_pulp_content_url }}/centos/8-stream/cloud/x86_64/openstack-wallaby/{{ stackhpc_pulp_repo_centos_stream_8_openstack_wallaby_version }}"
+    client_cert: "{{ stackhpc_release_pulp_client_cert }}"
```

Comment:
You need to switch to username and password instead of certificates.

Reply:
I pushed this as an additional commit.

Reply:
Thanks @priteau